code/src/App.js

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './button.css'
+
```

Comment:
It's nice how you put in a component of button. I think this is good to when you need to troubleshoot the code if there is a problem with it.